model/repo.go

```diff
@@ -67,4 +67,6 @@ type RepoPatch struct {
 	AllowDeploy  *bool   `json:"allow_deploy,omitempty"`
 	AllowTag     *bool   `json:"allow_tag,omitempty"`
 	BuildCounter *int    `json:"build_counter,omitempty"`
+	Name         *string `json:"name",omitempty`
```

Comment:
You're right it should be "name,omitempty" I'm not sure why it worked while testing but I'll fix and push